### ISSUE TITLE
[feature request] Power Management and "Connection to Dock failed"

### ISSUE THREAD

**Issue by alexthegreat100** [unfoldedcircle/bug-reports#295]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I think that most of the error messages that users see in everyday life have to do with the connection to the dock (#44).
So here is an idea on how to improve this.
On the one hand, many people want a long battery life and therefore switch off the "Prevent sleep mode" option. On the other hand, many people don't like to wait until the remote control is "ready for use" and therefore switch the function back on (#190).
There was already a request for sleep mode to be deactivated only when an activity is active (#281). I think that makes perfect sense.
Would it now be possible to maintain the connection to the dock via Bluetooth LE (or similar) in sleep mode so that commands can at least be sent or the devices can at least "see" each other?
This would result in slightly higher energy consumption while an activity is running, but would ensure a stable connection. And in sleep mode, the energy consumption would not drop quite as much, but a stable connection would also be guaranteed.

**Comment by henrikwidlund** [comment]
This sounds nice. It would also be helpful if the remote checks if connection is established, if not established try a couple of times before displaying the error.

**Comment by kennymc-c** [comment]
Yes, it makes no sense to that you can start an activity if not all needed integrations are connected. At least after waking up from sleep there should be a reconnection timeout until the activity changes to error status.
Maybe grey out the activity to not accidentally turn on already connected devices that you later have to manually turn off to not get state problems when rerunning the on sequence.
